Directory.Build.props

```diff
@@ -66,17 +66,19 @@
   <PropertyGroup>
 
     <VersionPrefix >$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
+    <!--
     <VersionSuffix>local-$(Configuration)-$([System.DateTime]::Now.ToString(yyyyMMdd))</VersionSuffix>
```

Comment:
This line was very, very handy for `TF_BUILD` == `''` cases, i.e. local machine builds.  

This allowed me to get reliably named packages on the local machine that were distinct from anything else on the nuget feeds. I found this critical when debugging across DiffSharp --> TorchSharp builds (one thing .NET PackageReference doesn't do well is a really easy idiot-proof way to pick up local builds of things).

Anyway it's ok, I haven't needed to debug like that in a long time